contracts/hardhat.config.ts

```diff
@@ -31,7 +31,29 @@ const config: HardhatUserConfig = {
     path: abigen.bytecodeExporterPath,
     runOnCompile: true,
     clear: true,
+  },
+  namedAccounts: {
+    deployer: { // Addressed used for deploying.
+        default: 0,
+    },
+    sequncer:{ // For management contract.
+        default: 1,
+    },
   }
 };
 
+try {
+  config.networks = JSON.parse(fs.readFileSync('config/networks.json', 'utf8'));
+} catch (e) {
+  console.log(`Failed parsing "config/networks.json" with reason - ${e}`);
+}
+
+try {
+  if (process.env.NETWORK_JSON != null) {
+    config.networks = JSON.parse(process.env.NETWORK_JSON!!);
+  }
+} catch (e) {
+  console.log(`Unable to parse networks configuration from environment variable. Reason is ${e}`);
+}
```

Comment:
Is reading from env and from file a requirement ?
Could the networks.json file be described in this file ?

Reply:
I've had the networks json described in here, but it felt like it was bloating the file. When default configs for testnet, simulation and so on are added at least.

Reply:
Cool, got it. What about the reading from env and file ? is the file read connected with the docker volume ?